ament_mypy/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ament_mypy</name>
+  <version>0.7.3</version>
+  <description>Support for mypy static type checking in ament.</description>
+  <maintainer email="ted.kern@canonical.com">Ted Kern</maintainer>
+  <license>Apache License 2.0</license>
+  <author email="ted.kern@canonical.com">Ted Kern</author>
+
+  <exec_depend>ament_lint</exec_depend>
```

Comment:
I don't see any usage of this package. Why was the dependency added?

Reply:
Removed, I think I was quick copying from flake8 and I grabbed a preceding line on my clipboard

Reply:
My mistake, fixed in 3a85e9f